src/sources/socket/mod.rs

```diff
@@ -227,7 +225,7 @@ mod test {
 
     #[test]
     fn tcp_continue_after_long_line() {
-        let (tx, rx) = mpsc::channel(10);
+        let (tx, rx) = Pipeline::new_test();
```

Comment:
I notice some of these had differing sizes in the channel creation, but `new_test` uses just one. Will that be an issue?

Reply:
It doesn't seem to be an issue. A lot of these were just guesses at a reasonable default, but if the test depends on a specific size we can switch back to specifying.